src/test/ui/issues/issue-72574-1.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    let x = (1, 2, 3);
+    match x {
+        (_a, _x @ ..) => {}
+        _ => {}
+    }
+}
+//~^^^^ ERROR `_x @` is not allowed in a tuple
+//~^^^^^ ERROR mismatched types [E0308]
```

Comment:
`|` continues checking on the same line as the check above.
```suggestion
//~| ERROR mismatched types [E0308]
```

-----

That being said, if you change lowering so as it won't emit the mismatched types error, you'll have to remove this.

Reply:
Will address these error annotations locally once I have a look at the errors emitted with the changes that you recommended.

Reply:
You are correct.  When I make the edit that you suggested in https://github.com/rust-lang/rust/pull/72677#discussion_r431964600, this goes away

Reply:
https://github.com/rust-lang/rust/pull/72677/commits/e73ac086c54084dced1916d062267be4058b257c

Reply:
Examples of these error annotations are now available in the rustc-dev-guide: https://rustc-dev-guide.rust-lang.org/tests/adding.html#error-annotation-examples